cellprofiler/gui/cpframe.py

```diff
@@ -857,6 +857,7 @@ def __add_menu(self):
         # ID_HELP_MODULE is used in _both_ button contexts and menu contexts,
         # so it needs event bindings for either type
         self.Bind(wx.EVT_MENU, self.__on_help_module, id=ID_HELP_MODULE)
+        self.Bind(wx.EVT_BUTTON, self.__on_help_module, id=ID_HELP_MODULE)
```

Comment:
I'm interested about this change - why was the `EVT_BUTTON` binding needed here for help as well?

Reply:
It looks like the help module functions were rewritten not too long ago. All of them are present as links in the menu, but this module is also called by the module help button within the main GUI pane. Without a button bind the help button does not work.